internal/app/machined/pkg/controllers/network/address_merge_test.go

```diff
@@ -152,7 +152,7 @@ func (suite *AddressMergeSuite) TestMerge() {
 	suite.assertAddresses(
 		[]string{
 			"lo/127.0.0.1/8",
-			"eth0/10.0.0.35/32",
+			"eth0/10.0.0.1/8",
```

Comment:
I guess this was a bug that got fixed?

Reply:
Yes, I did lots of runs with count 100 to find it )